USEFUL_RESOURCES.md

```diff
@@ -7,6 +7,22 @@
 * [Codeforces](https://codeforces.com/)
 * [Hackerearth](https://www.hackerearth.com/challenges/)
 
+### [GeeksForGeeks](https://www.geeksforgeeks.org/) - Best place to work on all the theoritical subjects including Database Management System, Data Structures, Data Algorithms, Object Oriented Programming, Operating Systems and many more.
+
+#### Benefits - 
```

Comment:
@Kritimakaushal  Thanks for contributing, but I think let's keep it simple, the `benefits` part seems unnecessary. Could you please remove that. The rest are fine ig

Reply:
Sure @Vishruth-S . Doing that.

Reply:
So I need to make a pull request again right? @Vishruth-S

Reply:
@Vishruth-S The updates are done.